pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -48,10 +49,13 @@ var _ framework.EnqueueExtensions = &VolumeRestrictions{}
 const Name = names.VolumeRestrictions
 
 const (
+	preFilterStateKey = "PreFilter" + Name
 	// ErrReasonDiskConflict is used for NoDiskConflict predicate error.
 	ErrReasonDiskConflict = "node(s) had no available disk"
-	// ErrReasonReadWriteOncePodConflict is used when a pod is found using the same PVC with the ReadWriteOncePod access mode.
-	ErrReasonReadWriteOncePodConflict = "node has pod using PersistentVolumeClaim with the same name and ReadWriteOncePod access mode"
```

Comment:
What happens if two pods are started with the same priority and referencing the same PVC with ReadWriteOncePod? What error will be surfaced to users when one pod inevitably fails to schedule?